src/main/webapp/web.xml

```diff
@@ -166,6 +166,24 @@
         <url-pattern>/login</url-pattern>
     </servlet-mapping>
 
+    <servlet>
+        <servlet-name>logout</servlet-name>
+        <servlet-class>internet.store.controllers.usercontroller.LogoutController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>logout</servlet-name>
+        <url-pattern>/logout</url-pattern>
+    </servlet-mapping>
+
+    <servlet>
+        <servlet-name>accessDenied</servlet-name>
+        <servlet-class>internet.store.controllers.usercontroller.LogoutController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>accessDenied</servlet-name>
+        <url-pattern>/user/accessDenied</url-pattern>
```

Comment:
```suggestion
        <url-pattern>/user/access-denied</url-pattern>
```

Reply:
Noticed accessDenied in Bohdan's video but your solution is nicer